src/main/java/org/AuthenticateDbHandler.java

```diff
@@ -0,0 +1,209 @@
+package org;
+
+import java.security.Key;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.util.Date;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import javax.crypto.spec.SecretKeySpec;
+import javax.xml.bind.DatatypeConverter;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.SignatureException;
+import oracle.jdbc.OraclePreparedStatement;
+
+public class AuthenticateDbHandler {
+	
+	Connection con;
+    public AuthenticateDbHandler(){
+        con = null;
+        try{
+        	DatabaseConnectionPool dbpool = DatabaseConnectionPool.getInstance();       	
+        	con = dbpool.getConnection();
+        }catch(Exception e){        }
+    }
+	
+	/*
+	 * WARNING.
+	 * This method is used to create a secret key for authentication
+	 * purposes. It ONLY returns the secret key as a String.
+	 * This secret key has to then be stored in the database table
+	 * 
+	 */
+	/*public String generateSecretKey(){
+		// create new key
+    	SecretKey secretKey = null;
+    	try {
+    	secretKey = KeyGenerator.getInstance("AES").generateKey();
+    	} catch (NoSuchAlgorithmException e) {
+    	e.printStackTrace();
+    	}
+    	
+    	// get base64 encoded version of the key
+    	String encodedKey = Base64.getEncoder().encodeToString(secretKey.getEncoded());
+    	return encodedKey;
+	}*/
+    
+    public Employee login(String username, String password){
+    	OraclePreparedStatement stmt = null;
+    	try {
+            stmt = (OraclePreparedStatement) con.prepareStatement(
+            		"select "
+            		+ "ID,"
+            		+ "NAME,"
+            		+ "COMPANIES_EMPLOYEE_ID,"
+            		+ "COMPANIES_ID,"
+            		+ "MANAGER,"
+            		+ "SUPER_ADMIN "
+            		+ "from employees where name = ? and web_password = ? ");
```

Comment:
The only other places I found were in the Employee class, the variable doesn't seem to be used anywhere else though